harmony/harmonydb/sql/20250422-msg-wait-timestamp.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE message_waits
+    ADD COLUMN added_at timestamptz NOT NULL DEFAULT TIMEZONE('UTC', NOW());
```

Comment:
The waits table holds historic data, we want a fancy conditional index on `created_at` + `where executed_tsk_cid is not null`

Reply:
https://www.postgresql.org/docs/current/indexes-partial.html

Reply:
Forgot that we don't clean up this one